edx_proctoring/api.py

```diff
@@ -1219,6 +1219,55 @@ def _get_email_template_paths(template_name, backend):
     return [base_template]
 
 
+def reset_practice_exam(exam_id, user_id):
+    """
+    Resets a completed practice exam attempt back to the created state.
+    """
+    log_msg = (
+        u'Resetting practice exam {exam_id} for user {user_id}'.format(
+            exam_id=exam_id,
+            user_id=user_id,
+        )
+    )
+    log.info(log_msg)
+
+    exam_attempt_obj = ProctoredExamStudentAttempt.objects.get_exam_attempt(exam_id, user_id)
+    exam = get_exam_by_id(exam_id)
```

Comment:
#nit I would define `exam` variable after checking that `exam_attempt_obj` is not None to avoid database requests in the worse case when the object is not found.